source/documentation/contributing_to_docs/contributing.adoc

```diff
@@ -0,0 +1,167 @@
+
+== Version management
+Most of the content applies to oVirt and Red Hat Virtualization. While a large
+amount of content is reused for both products, some information
+applies to only one or the other. Content inclusion and exclusion is managed
+by using a combination of `include` statements, `ifdef/endif` statements, and attributes
+in individual files.
+// by specifying distributions in the
+//`&#95;topic&#95;map.yml` file or by using `ifdef/endif` statements in individual files.
+
+////
+While it is _possible_
+to accomplish this solely with Git branches to maintain slightly different
+versions of a given topic, doing so would make the task of maintaining internal
+consistency extremely difficult for content contributors.
+
+////
+
+=== Conditional text between products
+oVirt documentation uses AsciiDoc's `ifdef/endif` and `ifeval/endif` macros to conditionalize
+and reuse content, down to the single-line level.
+
+The supported distribution attributes used with the oVirt build mechanism
+are:
+
+* _ovirt-doc_
+* _rhv-doc_
+
+Here is an example of this concept in use:
+
+----
+This line is not conditionalized, and will appear for both oVirt and RHV.
+
+\ifdef::ovirt-doc[]
+This line will only appear for oVirt.
+\endif::[]
+
+ifdef::rhv-doc
+This line will only appear for Red Hat Virtualization.
+\endif::[]
+
+----
+
+[NOTE]
+====
+While the `ifdef/endif` blocks have no size limit, do not use them to
+to conditionalize the text of an entire file within that file. Instead, use `include` statements to include the file in the top-level assembly, that is, the `index.adoc` file, or use `ifdef` macros around the `include` statement if the assembly is shared between oVirt and RHV.
+====
+
+== Release branches
+
+////
+With the combination of conditionalizing content within files with
+`ifdef/endif` and conditionalizing whole files in the `&#95;topic&#95;map.yml`
+file, the `main` branch of
+this repository always contains a complete set of documentation for all
+OpenShift products. However, when and as new versions of an OpenShift product
+are released, the `main` branch is merged down to new or existing release
+branches. Here is the general naming scheme used in the branches:
+
+* `main` - This is our *working* branch.
+* `enterprise-N.N` - OpenShift Container Platform support releases. The docs
+for OpenShift Online and OpenShift Dedicated are based on the appropriate
+`enterprise-N.N` branch.
+
+On a 12-hourly basis, the documentation web sites are rebuilt for each of these
+branches. This way the published content for each released version of an
+OpenShift product will remain the same while development continues on the
```

Comment:
suggestion "remains" -- it's a fact, so the present simple is accurate, as well as being preferred.

Reply:
Fixed, but this section is commented out and only applies to OpenShift.